build-config/resources/Ballerina.toml

```diff
@@ -10,13 +10,19 @@ license = ["Apache-2.0"]
 distribution = "2201.2.0"
 
 [[platform.java11.dependency]]
+groupId = "io.ballerina.stdlib"
+artifactId = "os-native"
+version = "@toml.version@"
 path = "../native/build/libs/os-native-@project.version@.jar"
 
 [[platform.java11.dependency]]
 groupId = "io.ballerina.stdlib"
 artifactId = "io-native"
-version = "@io.version@"
+version = "@io.native.version@"
```

Comment:
```suggestion
version = "@io.version@"
```

Reply:
Fixed in [53be925](https://github.com/ballerina-platform/module-ballerina-os/pull/442/commits/53be925a6e325eb5b03d6aee9b518dcaa95425f0)